src/heif_context.h

```diff
@@ -202,6 +218,20 @@ namespace heif {
 
     void write(StreamWriter& writer);
 
+    class InternalReader {
```

Comment:
How about "ReaderInterface" instead of "InternalReader" and "Reader_Memory" / "Reader_File" instead of InternalMemoryReader?

Reply:
Renamed the classes.